openedx/core/djangoapps/credentials/signals.py

```diff
@@ -89,11 +141,19 @@ def handle_grade_change(user, course_grade, course_key, **_kwargs):
     """
     Notifies the Credentials IDA about certain grades it needs for its records, when a grade changes.
     """
-    send_grade_if_interesting(user, course_key, None, None, course_grade.letter_grade, course_grade.percent)
+    send_grade_if_interesting(
+        user,
+        course_key,
+        None,
+        None,
+        course_grade.letter_grade,
+        course_grade.percent,
+        verbose=_kwargs.get('verbose', False)
```

Comment:
The reason an underscore is used in kwargs is for linting purposes. So that the linter won't complain about it not being used. If it is being used, we can drop the leading underscore.